<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">border: 5px solid rgb(0, 0, 0) ;
border-radius:20px;
background-color: qlineargradient(spread:pad, x1:0.409318, y1:0.636, x2:0.432, y2:0.0454545, stop:0 rgba(59, 123, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0,0,0,0%);
border:0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../assets/icono.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;Arial&quot;;
background-color:rgba(0,0,0,0%);
border:none;</string>
       </property>
       <property name="text">
        <string>Usuario</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="users">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
font: 75 12pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);
border:0px;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ingrese su correo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>260</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
font: 75 12pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);
border:0px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ingrese su contraseña</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_ingresar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:3px solid #ffffff;
border-radius:20px;
background-color: rgb(0, 255, 0);
font: 87 12pt &quot;Segoe UI Black&quot;}
QPushButton:hover{
border:4px solid #000000;
border-radius:20px;
background-color: rgb(255, 255, 0);
font: 87 12pt &quot;Segoe UI Black&quot;}</string>
       </property>
       <property name="text">
        <string>Iniciar sesión</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_close">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>452</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(83, 152, 255);
border:1px solid  rgb(83, 152, 255);
font: 87 10pt &quot;Segoe UI Black&quot;}
QPushButton:hover{
border:1px solid  rgb(83, 152, 255);
background-color:white;
font: 87 10pt &quot;Segoe UI Black&quot;}</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>400</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar{
background-color: rgba(0,0,0,0%);
border-radius:25px;
border: 1px solid #00007f;}
QProgressBar::chunk {
background-color: qlineargradient(spread:pad, x1:0.443545, y1:0.517, x2:0.989, y2:0.5, stop:0 rgba(255, 59, 250, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:25px;}</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>220</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;Arial&quot;;
background-color:rgba(0,0,0,0%);
border:none;</string>
       </property>
       <property name="text">
        <string>Contraseña</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="usuario_incorrecto">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>210</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0%);
color: rgb(255, 0, 0);
border:0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="contrasena_incorrecta">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>300</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0%);
color: rgb(255, 0, 0);
border:0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="cargando">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>410</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color: rgba(0,0,0,0%);
font: 12pt &quot;Segoe Print&quot;;
border:0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
